docs/volunteering/teams.md

```diff
@@ -0,0 +1,55 @@
+# Volunteer Teams
+
+Student Robotics has 2 teams which organise the operations regarding the wider competition programme. Each team is organised by its respective committee.
+
+- [Competition Team](https://opsmanual.studentrobotics.org/annual-robotics-competition/competition-team)
+- [Kit Team](https://opsmanual.studentrobotics.org/annual-robotics-competition/kit-team)
+
+## 2020
```

Comment:
Is this the calendar year 2020 or the SR year 2020? It looks like it's meant to be the SR year (which the competition team uses), however we should probably use "SR2020" to clarify that. However of course the Kit Team uses different timings, so using SR years there might be confusing.

Are there other ways of presenting this information which avoids this conflict? (If you've considered some but this still turns out best. please do say!)

Reply:
I'd intended this to be SR years, so yeah I think explicitly noting that is probably a good idea.

AFAIK, the Kit Team tenure falls conveniently around 2 competition cycles, so duplicating the list for the SR year is still correct. Thereby showing who was on the team for a given year, which is still useful information.